app/page/template/content/collection.htm

```diff
@@ -1,119 +1,79 @@
 <div id="collection"
-     class="collection content-wrapper"
+     class="collection content"
      data-bind="with: $root.collection, removed_from_view: $root.collection.removed_from_view">
 
   <!-- ko if: conversation_et() -->
 
     <div class="content-titlebar">
-
-      <span class="content-titlebar-items-center" data-bind="text: conversation_et().display_name()"></span>
-
-      <div class="content-titlebar-items-right">
+      <div class="content-titlebar-items-left">
         <span class="content-titlebar-icon icon-close" data-bind="click: click_on_back_button"></span>
       </div>
-
+      <span class="content-titlebar-items-center" data-bind="text: conversation_et().display_name()"></span>
     </div>
 
-    <div class="content-list collection-list">
+    <div class="content-list-wrapper">
+      <div class="content-list" data-bind="antiscroll: true">
 
-      <!-- ko ifnot: no_items_found() -->
-
-        <!-- ko if: images().length -->
-          <section class="collection-section" data-uie-name="collection-section-image">
-            <header>
-              <span class="collection-header-icon icon-library"></span>
-              <span class="label-bold-xs">Pictures</span>
-              <!-- ko if: images().length > 10 -->
+        <!-- ko ifnot: no_items_found() -->
+          <!-- ko if: images().length -->
+            <section class="collection-section" data-uie-name="collection-section-image">
+              <header>
+                <span class="collection-header-icon icon-library"></span>
+                <span class="label-bold-xs">Pictures</span>
+                <!-- ko if: images().length > 12 -->
```

Comment:
Missing appropriate end tag. Looks like the whole section will only be shown if more than 12 items.